Plugins/CoreAPI/server/plugin.js

```diff
@@ -0,0 +1,42 @@
+import { types, resolvers, loaders } from "./graphql";
+import PenPal from "meteor/penpal";
+import _ from "lodash";
+
+const settings = {
+  configuration: {
+    schema_root: 'CoreAPIConfiguration',
+    getter: 'getCoreAPIConfiguration',
+    setter: 'setCoreAPIConfiguration'
+  },
+  datastores: [
+    {
+      name: "Projects"
```

Comment:
If we only have one field on each of these objects, do we need to make them an object or just an array of strings? Are we going to need other fields on them in the future?

Reply:
I imagine we may eventually allow the user to specify "indexed" or something for optimization purposes, I can change it if you want (also the example of what you wanted that you showed me was objects not string array ;) )

Reply:
No leaving as objects is fine. Was just curious on thought process (and clearly forgot my own contributions to that)